ci/dev/vscode.patch

```diff
@@ -1049,6 +1024,21 @@ index 0000000000000000000000000000000000000000..0d2e93edae2baf34d27b7b52be0bf496
 +		}
 +	}
 +
++	async $fetchExtension(extensionUri: UriComponents): Promise<VSBuffer> {
```

Comment:
Why do we have to implement this?

Reply:
I made a comment on https://github.com/cdr/code-server/pull/2204/commits/f20f7ac166b40b7420a6114968a3506362280b54 but the gist is that the worker is now in a separate origin so it can't fetch the extension (cross origin domain issues) and even if you resolve that by setting the appropriate headers it still doesn't work because it doesn't send the cookie. So instead I have it asking the main thread to fetch the extension. Since this mechanism already exists for the extension to communicate with the main thread (for example to make file read calls) I figured it was a decent solution.

Reply:
I should comment this in the code and not just in the commit.